src/units/active_ability_list.cpp

```diff
@@ -0,0 +1,352 @@
+/*
+	Copyright (C) 2006 - 2022
+	by Dominic Bolin <dominic.bolin@exong.net>
```

Comment:
It's one thing to put Dave in by default, as he's the founder of the project, but filling in anyone else as the copyright seems wrong.

Reply:
This file was split out from ``abilities.cpp`` (formerly ``unit_abilities.cpp``), which was originally attributed to Dominic. And even though he hasn't been active for over 15 years, it doesn't seem entirely wrong to attribute this file to him.

Frankly there have been lots of changes to that file since then, but no full rewrite (as was the case in 72b0cf9bdbf).

Reply:
I'll probably remove it , i just copied the file and didn't  think about this.